### DISCUSSION TITLE
Zooming feature

### DISCUSSION THREAD

**Discussion by timothycuenat** [recharts/recharts#4634] [Ideas]
Hello
This library is very good and easy to use. It is missing just the zooming and panning feature. With mouse click and mous scroll would it be good.
I hope it could be one day added. 
Thanks

**Comment by ckifer** [comment]
Hi!

Does this help at all?

https://recharts.org/en-US/examples/HighlightAndZoomLineChart

It may not be built in but you can get close pretty easily
